ikarus/python/dirichletvalues/dirichletvalues.hh

```diff
@@ -57,54 +61,118 @@ void registerDirichletValues(pybind11::handle scope, pybind11::class_<DirichletV
   using Intersection = typename Basis::GridView::Intersection;
 
   pybind11::module scopedf = pybind11::module::import("dune.functions");
+
   typedef Dune::Python::LocalViewWrapper<Basis> LocalViewWrapper;
   auto includes = Dune::Python::IncludeFiles{"dune/python/functions/globalbasis.hh"};
-  auto lv       = Dune::Python::insertClass<LocalViewWrapper>(
-                scopedf, "LocalView",
-                Dune::Python::GenerateTypeName("Dune::Python::LocalViewWrapper", Dune::MetaType<Basis>()), includes)
-                .first;
+  auto lv_      = Dune::Python::insertClass<LocalViewWrapper>(
+                 scopedf, "LocalView",
+                 Dune::Python::GenerateTypeName("Dune::Python::LocalViewWrapper", Dune::MetaType<Basis>()), includes)
+                 .first;
 
   cls.def(pybind11::init([](const Basis& basis) { return new DirichletValues(basis); }), pybind11::keep_alive<1, 2>());
 
-  // Eigen::Ref needed due to https://pybind11.readthedocs.io/en/stable/advanced/cast/eigen.html#pass-by-reference
-  cls.def("fixBoundaryDOFs",
-          [](DirichletValues& self, const std::function<void(Eigen::Ref<Eigen::VectorX<bool>>, int)>& f) {
-            auto lambda = [&](BackendType& vec, const MultiIndex& indexGlobal) {
-              // we explicitly only allow flat indices
-              f(vec.vector(), indexGlobal[0]);
-            };
-            self.fixBoundaryDOFs(lambda);
-          });
+  cls.def_property_readonly("container", &DirichletValues::container);
```

Comment:
Why as read only?

Reply:
Because for a `def_property` i need a setter and a getter i think, also I think its cleaner to work with `fixIthDegree()`

Reply:
Also the c++ method `container()` returns as const ref